src/hooks/command_not_found/api-action.ts

```diff
@@ -1,31 +1,8 @@
 import chalk from "chalk";
 import { Hook } from "@oclif/config";
-import SdkQuery from "../../commands/sdk/query";
 
 /**
- * Hooks that run the corresponding API action with sdk:query.
```

Comment:
Actually I would like to keep the inference part which is quite handy when executing API action.  
The automatic inference was more about `kourou document:get` => `kourou api document:get` but then it's more practical to enter common API argument without the `-i` or `-c` (etc) flags

Reply:
@OlivierCavadenti

Reply:
I revert this file but I don't understand what to do with this issue at the end : "the automatic inference of commands as API action should be removed"

Reply:
I don't remember why we wrote this when we did the workshop for Kourou v1 but I'm using this inference every day and it's quite handy so if we really want to remove it then I want to discuss first